packages/components/src/internal/components/entities/ParentEntityEditPanel.tsx

```diff
@@ -108,11 +115,16 @@ export class ParentEntityEditPanel extends Component<Props, State> {
         this.setState({
             currentParents: originalParents,
             loading: false,
+            editing: this.props.editOnly,
             originalParents,
-            parentTypeOptions,
+            parentTypeOptions: List<IEntityTypeOption>(parentTypeOptions.sortBy(p => p.label, naturalSort).toArray()),
```

Comment:
would the `naturalSortByProperty` helper work here?

Reply:
yup. thanks.